bindings/swift/native/Cargo.toml

```
@@ -0,0 +1,22 @@
+[workspace]
+
+[package]
+name = "iota-wallet-swift-binding"
+version = "0.1.0"
+authors = ["Thoralf Müller <thoralf.mueller@iota.org", "Pasquale Ambrosini <pasquale.ambrosini@gmail.com"]
```

Perhaps follow what we've been doing.

```suggestion
authors = ["IOTA Stiftung"]
```